Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -1126,6 +1136,17 @@ namespace AzToolsFramework
         m_selectedEntityIds.clear();
         GetSelectedEntities(m_selectedEntityIds);
 
+        // Check if any of the selected entities are marked as read only
+        m_entityIsReadOnly = false;
```

Comment:
I think the name for this member variable is a little misleading. Maybe something like `m_selectionContainsReadOnlyEntity` would be more descriptive?

Reply:
Yeah when I started I forgot about selecting multiple entities, so my logic does account for that but I didn't rename the variable, I will do that :)